server/channels/app/imports/import_validators.go

```diff
@@ -706,44 +706,74 @@ func isValidEmailBatchingInterval(emailInterval string) bool {
 		emailInterval == model.PreferenceEmailIntervalHour
 }
 
-// isValidGuestRoles checks if the user has both guest roles in the same team or channel.
+// isValidGuestRoles checks if the user has guest roles consistently across system, team, and channel levels.
 // at this point we assume that the user has a valid role scheme.
 func isValidGuestRoles(data UserImportData) bool {
```

Comment:
Should this method return a proper `error` instead of a boolean? At the moment, it is kind of hard to understand what is going wrong from an admin perspective.

Reply:
resolved this with the latest changes pushed.